images/views.py

```diff
@@ -1,42 +1,12 @@
-from django.db.models import Q
-from rest_framework import generics
-from rest_framework import permissions
+from rest_framework import generics, permissions
 
-from .exceptions import (
```

Comment:
I think it is also necessary to remove: https://github.com/tolstoyevsky/cusdeb-api/pull/10/files#diff-670ee35a83fc2c6bebbae99a21e7302b